### PR TITLE
[fix-sf-connection] Only require active SF connection for org creation and deletion

### PR DESCRIPTION
**Pull request by wlonk**
See https://trello.com/c/qDdlVfF2/90-bug-only-require-active-sf-connection-for-org-creation-and-deletion

cat

### CODE REVIEW

**Comment by wlonk** [comment]
@davisagli Can you confirm that this is the ID we mean to get? It is the same as the one in the org config, but I'm not certain and would appreciate a more informed perspective.

**Comment by wlonk** [comment]
[Bug: Only require active SF connection for org creation and deletion.](https://trello.com/c/qDdlVfF2/90-bug-only-require-active-sf-connection-for-org-creation-and-deletion)

**Comment by davisagli** [comment]
@wlonk What are you aiming to do with it? Our conclusion this morning was that we don't need to enforce that the user is still connected to the same SF org that they were when a scratch org was created. Which I think means we don't need this. But maybe I'm forgetting something else we need it for?

**Comment by jgerigmeyer** [comment]
@davisagli We discussed still needing this info for deleting a scratch org. (See the Trello card description for my mtg notes -- correct me if they're wrong: https://trello.com/c/qDdlVfF2)

**Comment by davisagli** [comment]
@jgerigmeyer oh yes, thanks for the reminder. In that case @wlonk we should probably store the username of the user who was logged in, not the org id. Depending on how the permissions are configured in the dev hub org, a user may not have access to delete a different user's scratch orgs.

**Comment by jgerigmeyer** [comment]
[Only refresh unsaved_changes on view list/retrieve if org is owned by requesting user.](https://trello.com/c/14s19wwg/108-only-refresh-unsavedchanges-on-view-list-retrieve-if-org-is-owned-by-requesting-user)

**Comment by jgerigmeyer** [comment]
Requesting code review from @wlonk (and @davisagli as he has time).

New modal appears if:

- Logged in as same GH user that created org
- Connected to SF, but different SF user than org creator
- Clicked button to delete org

Screenshot 2019-11-05 13 59 29

**Comment by wlonk** [comment]
@jgerigmeyer looked over it, looks good.

**Comment by jgerigmeyer** [comment]
> Looks good to me. But the test failed?

Yeah, looking at that now.

File: hooks/pre-commit [hunk 1]
```diff
@@ -19,6 +19,12 @@ function dce {
     RUNNING=`is_running`
     case "$RUNNING" in
         "running")
+            # XXX: I have discovered a truly marvelous bug which this margin is
+            # too small to contain. But in seriousness: if the container is
+            # "running" because of `docker-compose run` instead of
+            # `docker-compose up`, then `docker-compose exec` will fail. For my
+            # server-side purposes, I tend to not ever run `up`, only `run
+            # bash`, and so this reliably fails. I should fix this!
```
Comment (jgerigmeyer) [inline hooks/pre-commit]: @wlonk Related to this, I am experiencing some other bugs with this script. Captured in a new Trello board and card: https://trello.com/c/lrRzXLdP

**Pull request merged**